test/e2e/performance_test.go

```diff
@@ -25,30 +25,39 @@ import (
 	"golang.org/x/exp/rand"
 	"k8s.io/apimachinery/pkg/util/wait"
 
-	v1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 	networkv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	seed                            uint64 = 0xA1E47 // Use a specific rand seed to make the generated workloads always same
-	performanceTestAppLabel                = "antrea-performance-test"
+	perfTestAppLabel                       = "antrea-perf-test"
 	podsConnectionNetworkPolicyName        = "pods.ingress"
 	workloadNetworkPolicyName              = "workloads.ingress"
-	abImage                                = "antrea/apache-bench"
+	perfToolImage                          = "antrea/perftool"
 	nginxImage                             = "nginx"
-	abContainerName                        = "apache-bench"
+	perftoolContainerName                  = "perftool"
```

Comment:
you use `perfTool` for the image name (`perfToolImage`), but `perftool` for the container name (`perftoolContainerName`)

Reply:
Thanks, changed to `perftool`.